menu/cbs/menu_cbs_ok.c

```diff
@@ -2532,6 +2585,11 @@ static int action_ok_playlist_entry_collection(const char *path,
    if (!playlist_entry_path_is_valid(content_path))
       goto error;
 
+#ifdef HAVE_ENTRYSTATES
+   if (!playlist_entry_state_is_valid(content_state))
```

Comment:
Surely this is a mistake...? We don't want to block content loading if there is no entry state...

Reply:
Its gone ;)